app/scripts/modules/ecs/src/serverGroup/configure/wizard/networking/networkingSelector.component.html

```diff
@@ -57,6 +57,26 @@
     </div>
   </div>
 
+  <div ng-if="$ctrl.command.networkMode == 'awsvpc'">
+    <div class="form-group">
+      <div class="col-md-4 sm-label-right">
+        <b>Associate Public IP Address</b> <help-field key="ecs.publicip"></help-field>
+      </div>
+      <div class="col-md-1 radio">
+        <label>
+          <input type="radio" ng-model="$ctrl.command.associatePublicIpAddress" ng-value="true" id="associatePublicIpAddressTrue"/>
```

Comment:
A checkbox is more common UX for binary choices, especially for yes/no, true/false choices. It's up to you though.